src/head.rs

```diff
@@ -0,0 +1,27 @@
+use bevy::{math::DVec2, prelude::*, utils::Uuid};
+
+use bevy_vello::prelude::*;
+
+#[derive(Default, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct ShapeId(Uuid);
+
+#[derive(Component, Default, Copy, Clone)]
+pub struct Head {
+    pub shape_id: ShapeId,
+
+    pub scale: f32,
+    pub offset: f32,
+    pub rotation_offset: f32,
+}
+
+#[derive(Resource)]
+pub struct Shapes {
+    pub scenes: std::collections::HashMap<ShapeId, &'static mut vello::Scene>,
+}
+
+pub trait VectorBorder {
+    /// translation of the chosen "apex"
+    fn border_translation(&self, time: f32) -> DVec2;
+    /// gradient of the tangent to the border
```

Comment:
```suggestion
    /// The tangent of the border at a specific `time` value.
```

Reply:
I think gradient should stay, i know since we return a float its obvious that were talking about the gradient but i think it would be better to have for the docs but if you still want me to change it let me know